### PR TITLE
Tk#116   create a paginary

### LINKED ISSUE #116: CREAR UN PAGINARIO / CREATE A PAGINARY
PROJECT TASK TEMPLATE

### EXPLANATION

En la vista añadir socio, encontraremos en la parte inferior de la pantalla un paginario con la  información del numero de socios con las opciones de antes y siguiente.

In the add member view, we will find at the bottom of the screen a paginary with the information of the number of members with the options before and next

### SCREENSHOT

Image

Image

### CHECKLIST

Cuando se abre la vista de añadir socio, debe haber en la parte de abajo un paginario para que el usuario pueda saber información del numero de miembros inscritos en la pagina. Este paginario debe tener las opciones antes y siguiente, para retroceder o avanzar en cada pagina. Su función sera  mostrar los socios añadidos con unos números que determina cuantos hay en una lista por números

When the add member view opens, there should be a paginary at the bottom of the page so that the user can see information about the number of members registered on the page. This paginary should have the options before and next, to go back or forward in each page.

### PR DESCRIPTION
**Pull request by JuanParraIV**
### Description

Please include a summary of the change.

fixes # (issue)
(116)
#### Screenshots

If applicable, add screenshots about your changes
image

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by Eghinner — CHANGES_REQUESTED** [review]
chrome_ol6JhiAzyt
should always be at the bottom of the page

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Eghinner** [comment]
Can you update this branch with `dev`. This will allow a test deployment. Then resolve conflicts
`git fetch -a`
`git pull origin dev --force`

File: components/Paginated/Paginated.module.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+.paginated-container {
+  align-items: center;
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  margin: 10px 50px;
+  max-height: 43px;
+  padding: 20px auto;
+  position: fixed;
```
Comment (Eghinner) [inline components/Paginated/Paginated.module.scss]:
```suggestion
  position: absolute;
  right: 0;
```
but you have also his father element relative. Read this [ref ](https://css-tricks.com/absolute-positioning-inside-relative-positioning/)

**Comment by JuanParraIV** [comment]
How?
i already use git fetch -a
git pull origin dev --force doesnt work

**Review by Eghinner — APPROVED** [review]

File: components/Paginated/Paginated.module.scss (hunk 1 above)
Comment (JuanParraIV) [reply components/Paginated/Paginated.module.scss]: ok

Comment (JuanParraIV) [reply components/Paginated/Paginated.module.scss]: only these?

File: pages/socios/index.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import UsersList from '@/views/Users/List'
 const Users: NextPage = () => {
   const { data, loading } = useQuery(USERS)
   const [CurrentPage, setCurrentPage] = useState(1)
-  const PartnersPerPage = 12
+  const PartnersPerPage = 2
```
Comment (Israel-Laguan) [inline pages/socios/index.tsx]: Please leave as 12

Comment (JuanParraIV) [reply pages/socios/index.tsx]: ok

**Comment by JuanParraIV** [comment]
Screenshot from 2022-09-03 09-14-55

**Comment by Eghinner** [comment]
> Screenshot from 2022-09-03 09-14-55

it seems that in fonts.scss there is a font that does not exist anymore, which I deleted

File: styles/fonts.scss [hunk 3]
```diff
@@ -53,17 +53,18 @@ $font-base-size: 16px;
...
 $font-base-variant: normal;
 
 @font-face {
-  font-family: 'Visby';
-  src: url('~/public/assets/fonts/VisbyCF-Regular.otf');
-  font-weight: 400;
+  font-family: 'Visby' serif;
   font-style: normal;
+  font-weight: 400;
+  src: url('../public/assets/fonts/VisbyRegular.otf');
```
Comment (Eghinner) [inline styles/fonts.scss]: This. Just delete it

**Comment by angelik0828** [comment]
## I am on the lookout to validate the changes in this PR I still can't see it

image

**Comment by Israel-Laguan** [comment]
> ### I am on the lookout to validate the changes in this PR I still can't see it

Please check the latest versions:
image

If you click the button "View deployment" you can find the latest version:
https://did-admin-panel-cjltlwgl8-azordev.vercel.app/

**Comment by angelik0828** [comment]
## It has been verified that the font is present in the side menu, check if you can use that font in the pager, currently the change is not applied. 

image

image

**Review by Israel-Laguan — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

**Pull request merged**